src/TaskLineRenderer.ts

```diff
@@ -140,6 +140,17 @@ async function taskToHtml(
         if (componentString) {
             if (component === 'description') {
                 componentString = GlobalFilter.removeAsWordFromDependingOnSettings(componentString);
+
+                if (renderDetails.layoutOptions?.hideTags) {
+                    // This matches a tag only when it's a complete word
+                    const tagRegExp = RegExp('(#+[a-zA-Z0-9(_)]{1,})', 'ug');
```

Comment:
That's interesting... It uses its own RegExp for identifying tags... What would that mean in terms of maintenance, like if the main one changed and this didn't, or vice versa?